core/lib/types/src/contract_verification/api.rs

```diff
@@ -147,6 +147,19 @@ pub struct VerificationIncomingRequest {
     pub is_system: bool,
     #[serde(default)]
     pub force_evmla: bool,
+    #[serde(flatten)]
+    pub evm_specific: VerificationEvmSettings,
+}
+
+/// Settings for EVM verification, used only if
+/// `SourceCodeData` is `SolSingleFile`.
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct VerificationEvmSettings {
+    #[serde(default)]
```

Comment:
Don't know whether it matters much, but since this is serialized back to JSON to store in Postgres, it could make sense to have `skip_serializing_if = "Option::is_none"` (or even put a `null` altogether if neither of fields is specified) in order to not bloat the table size.

Reply:
I like `skip_serializing_if` more, it sounds both more flexible and doesn't add custom handling to DAL.